api/tests/integration/infrastructure/repositories/membership-repository_test.js

```diff
@@ -156,6 +156,27 @@ describe('Integration | Infrastructure | Repository | membership-repository', ()
       // given
       const organization = databaseBuilder.factory.buildOrganization();
 
+      const user_1 = databaseBuilder.factory.buildUser({ lastName: 'Martin', firstName: 'George' });
+      const user_2 = databaseBuilder.factory.buildUser({ lastName: 'Luther', firstName: 'Zoe' });
+      const user_3 = databaseBuilder.factory.buildUser({ lastName: 'Luther', firstName: 'Arthur' });
+
+      const membership_1 = databaseBuilder.factory.buildMembership({ organizationRole: Membership.roles.ADMIN, organizationId: organization.id, userId: user_1.id });
+      const membership_2 = databaseBuilder.factory.buildMembership({ organizationRole: Membership.roles.MEMBER, organizationId: organization.id, userId: user_2.id });
+      const membership_3 = databaseBuilder.factory.buildMembership({ organizationRole: Membership.roles.MEMBER, organizationId: organization.id, userId: user_3.id });
+
+      await databaseBuilder.commit();
+
+      // when
+      const memberships = await membershipRepository.findByOrganizationId({ organizationId: organization.id, orderByName: true });
+
+      // then
+      expect(_.map(memberships, 'id')).to.deep.equal([membership_1.id, membership_3.id, membership_2.id]);
+    });
+
+    it('shoud order memberships by organizationRole, user.lastName and user.firstName', async () => {
```

Comment:
ce test a son nom de changé, mais aucune mention du rôle dans le corps du test...

Reply:
Et on a perdu un L dans "should"